.action_templates/jobs/tests.yaml

```diff
@@ -54,3 +54,7 @@ tests:
           distro: ubi
         - test-name: replica_set_mount_connection_string
           distro: ubi
+        - test-name: replica_set_mongod_port_change
+          distro: ubi
+        - test-name: replica_set_mongod_port_change_with_arbiters
```

Comment:
Curious why do we need 2 tests here, can we just run 1 e2e test which has both MongoDB deployment and arbiters?

Reply:
My thought process was that configuration with arbiters and without them is different, so we should have e2e tests to both scenarios. But if I think about it now it is unnecessary as arbiters functionality is just an extension, so one suit will suffice. Removed it.